src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php

```diff
@@ -578,4 +578,74 @@ public function testThrowTypeErrorWithInterface()
 
         $this->propertyAccessor->setValue($object, 'countable', 'This is a string, \Countable expected.');
     }
+
+
+    public function testAnonymousClassRead()
+    {
+        if (PHP_MAJOR_VERSION < 7) {
+            $this->markTestSkipped('Anonymous Classes are only supported on PHP7');
+            return;
```

Comment:
we prefer using `@requires php 7` annotations
this made me notice that this should be submitted (rebased+base branch changed) against the 3.3 branch